<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PharmaciesForm</class>
 <widget class="QDialog" name="PharmaciesForm">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>465</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>388</width>
    <height>465</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>388</width>
    <height>465</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pharmacies information</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>411</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QPushButton" name="moreInfo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Show more info</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>430</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sortButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>430</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Sort</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
